### ISSUE TITLE
[BUG] - @epicgames-ps/lib-pixelstreamingfrontend-ue5.5 is including EventEmitter, which causes build problems

### ISSUE THREAD

**Issue by leon** [EpicGames/PixelStreamingInfrastructure#328]
**UE Version:**
E.g. UE 5.5.0

**Frontend Version:**
E.g. UE5.5-0.3.2

**Problem component**
E.g. Frontend

**Description**
I have a very simple vite project that is importing
```typescript
import {
	Config,
	PixelStreaming,
} from '@epicgames-ps/lib-pixelstreamingfrontend-ue5.5';
```

I tried upgrading from v 2.1.0 to 0.3.2 but I'm now getting Module "events" has been externalized for browser compatibility. Cannot access "events.EventEmitter"...

The problem seems to be that `@epicgames-ps+lib-pixelstreamingcommon-ue5.5` `WebSocketTransport.js` is getting included in the frontend code and it uses nodes EventEmitter, which is a node only library.

**Steps to Reproduce:**
create a new vite project
use v0.3.2
try building

here is a repo that you can test out
https://github.com/leon/bug-pixelstreaming-eventemitter

**Expected behavior**
Should be able to build frontend using 5.5 v0.3.2

**Screenshots**

**Platform (please complete the following information):**
MacOS Sequoia

**Comment by lukehb** [comment]
Thanks for reporting, I will look into this today.

**Comment by lukehb** [comment]
Working on a PR - should have it fixed tomorrow: https://github.com/EpicGamesExt/PixelStreamingInfrastructure/pull/330

**Comment by lukehb** [comment]
If you try bumping to the latest (0.4.0) @epicgames-ps/lib-pixelstreamingfrontend-ue5.5 it should be fixed for you (I tested against your provided repo and it built).

**Comment by leon** [comment]
Hey,

Just wanted to mention that 0.4.0 is not tagged as the latest release on npm
https://www.npmjs.com/package/@epicgames-ps/lib-pixelstreamingfrontend-ue5.5?activeTab=versions

**Comment by mcottontensor** [comment]
Yeah thats because I had to push a fix for the 0.3.x version.
@lukehb You might be able to make it as latest, but I don't have direct perms.
